src/registration.rs

```diff
@@ -278,6 +300,17 @@ async fn ensure_user_exists(
 }
 
 pub async fn handle_user_registration(config: &crate::Config) -> Result<(), Error> {
+    // Create an admin user, we'll need it later.
```

Comment:
later for what?

Reply:
To unthrottle users.

Amended comment :)